zio-cli/shared/src/main/scala/zio/cli/Options.scala

```diff
@@ -67,6 +72,26 @@ sealed trait Options[+A] { self =>
   )(implicit ev: A <:< ((B, (C, (D, (E, (F, G))))))): Options[Z] =
     self.map(ev).map { case ((b, (c, (d, (e, (f, g)))))) => f0(b, c, d, e, f, g) }
 
+  final def fold[B, C, Z](
+    f1: B => Z,
+    f2: C => Z
+  )(implicit ev: A <:< Either[B, C], ctb: ClassTag[B], ctc: ClassTag[C]): Options[Z] =
```

Comment:
Can we remove the `ClassTag` here? There is no need to match on the type of the `Either`, since Scala knows what it is, with a little help (mapping using `ev` will solve the problem, e.g. `self.map(ev).map { case Left(b) => ... case Right(c) => ... }`).

Reply:
Of course using map with the evidence is much better! I did not see that. 
Let me remove those unnecessary implicit class tags.